R/scale-.R

```diff
@@ -1454,5 +1454,5 @@ trans_support_nbreaks <- function(trans) {
 }
 
 allow_lambda <- function(x) {
-  if (is_formula(x)) as_function(x) else x
+  if (is_formula(x, lhs = FALSE)) as_function(x) else x
```

Comment:
Wouldn't it be better to test if x is of class `call` instead of `formula`? This would guard us against any use of bquote, also ones that result in a one-sided formula

The biggest issue is that `is_formula` and `is_call` both return true for formulas and calls that are formulas, even though `class()` differs

Reply:
It depends. Do we want bquotes to never be interpreted as a lambda? If so, we can add a `&& !inherits(x, "call")` clause.

Reply:
I can't see any good reason why you'd use bquote to create a lambda